verification/io/other_defs.gobra

```diff
@@ -120,19 +120,19 @@ func (h IO_HF) Toab() IO_ahi {
 // TODO: move ToHF to another package
 // TODO: can we get rid of this assumption?
 // We ASSUME the existence of a function that computes the msgTerm of a path.HopField
-ghost
-decreases
-pure func ComputeMsgTerm(h path.HopField) IO_msgterm
-
-// TODO: should be easy to give a body to this now
-ghost
-ensures h.ConsIngress == 0 ==> res.InIF2 == none[IO_ifs]
-ensures h.ConsIngress != 0 ==> res.InIF2 == some(IO_ifs(h.ConsIngress))
-ensures h.ConsEgress  == 0 ==> res.EgIF2 == none[IO_ifs]
-ensures h.ConsEgress  != 0 ==> res.EgIF2 == some(IO_ifs(h.ConsIngress))
-ensures res.HVF == ComputeMsgTerm(h)
-decreases
-pure func ToHF(h path.HopField) (res IO_HF)
+// ghost
+// decreases
+// pure func ComputeMsgTerm(h path.HopField) IO_msgterm
+
+// // TODO: should be easy to give a body to this now
+// ghost
+// ensures h.ConsIngress == 0 ==> res.InIF2 == none[IO_ifs]
+// ensures h.ConsIngress != 0 ==> res.InIF2 == some(IO_ifs(h.ConsIngress))
+// ensures h.ConsEgress  == 0 ==> res.EgIF2 == none[IO_ifs]
+// ensures h.ConsEgress  != 0 ==> res.EgIF2 == some(IO_ifs(h.ConsIngress))
+// ensures res.HVF == ComputeMsgTerm(h)
+// decreases
+// pure func ToHF(h path.HopField) (res IO_HF)
```

Comment:
Are you really sure that these two functions will never be called from the IO spec?